analogchessV4.py

```diff
@@ -95,7 +95,13 @@ def __init__(self, x_pos, y_pos, color):
         self.start_y=self.y
         text_scale = 0.85
         self.letter = 'X'
-        self.font = pygame.font.SysFont("segoeuisymbol", int(diameter/8*640*text_scale))
+        
+        font_options = ["segoeuisymbol", "applesymbols"]

```

Comment:
Could you also add "DejaVuSans" for Ubuntu? It doesn't package the Segoe UI fonts